Firmware/LowLevel/include/config.h

```diff
@@ -0,0 +1,19 @@
+#ifndef _CONFIG_H
+#define _CONFIG_H
+
+#define BATT_ABS_MAX 28.7f
+#define BATT_ABS_Min 21.7f
```

Comment:
Regarding "_Min" yes, changed. Copied from main and didn't validated.

Aren't all our macro values defaults? So no I wouldn't do so, also because I only moved them from main.cpp.
I always try to let them as they have been.

Regarding the const vs. macro. I'm undecided with this. A const take the whole footprint as a normal variable. A macro is simply inline expanded during compilation (what you know for sure). However, here I try the same: If the author preferred to use macros, I follow em, except the other make more sense. But mixing up without reason, I wouldn't do.

Reply:
> Aren't all our macro values defaults?

No, `IMU_CYCLETIME` and many others are not. That's why I thought about indicating that these values aren't fixed, but can be changed later. How about simply renaming the file to "config_defaults.h"?

> A const take the whole footprint as a normal variable. A macro is simply inline expanded during compilation (what you know for sure).

Maybe the compiler also inlines `const`s and removes their footprint? Or maybe for a `static const`? Not sure.

But yeah, mixing up would be bad. All or nothing, so probably let it as-is.

Reply:
Yea "config_defaults.h" is fine. I'm bad with finding good names at all. Doesn't matter if function, method, class or file name :-(